bin/migrate_to_single_table.py

```diff
@@ -111,66 +132,23 @@ def index_record_to_new_table(self, batch_size=1000):
                 if not records:
                     break
 
-                records_to_insert = []
-
-                for record in records:
-                    hashes = self.get_index_record_hash(record.did)
-                    urls = self.get_urls_record(record.did)
-                    url_metadata = self.get_urls_metadata(record.did)
-                    acl = self.get_index_record_ace(record.did)
-                    authz = self.get_index_record_authz(record.did)
-                    alias = self.get_index_record_alias(record.did)
-                    metadata = self.get_index_record_metadata(record.did)
-                    records_to_insert.append(
-                        Record(
-                            guid=record.did,
-                            baseid=record.baseid,
-                            rev=record.rev,
-                            form=record.form,
-                            size=record.size,
-                            created_date=record.created_date,
-                            updated_date=record.updated_date,
-                            content_created_date=record.content_created_date,
-                            content_updated_date=record.content_updated_date,
-                            file_name=record.file_name,
-                            version=record.version,
-                            uploader=record.uploader,
-                            hashes=hashes,
-                            urls=urls,
-                            url_metadata=url_metadata,
-                            acl=acl,
-                            authz=authz,
-                            alias=alias,
-                            record_metadata=metadata,
-                        )
-                    )
-                while records_to_insert:
-                    try:
-                        self.session.bulk_save_objects(records_to_insert)
-                        self.session.commit()
-                        count += len(records_to_insert)
-                        self.logger.info(
-                            f"Done processing {count}/{total_records} records. {(count * 100)/total_records}%"
-                        )
-                        break
-                    except Exception as e:
-                        self.session.rollback()
-                        if "duplicate key value violates unique constraint" in str(e):
-                            records_to_insert = self.remove_duplicate_records(
-                                records_to_insert, e
-                            )
+                records_to_insert = self.get_info_from_mult_tables(records)
+
+                self.bulk_insert_records(records_to_insert)
```

Comment:
How are you handling errors in this new code? It looks like they don't re-raise from this function, so it just continues to run new batches. Is that intended? Are people expected to parse the logs for errors and rerun with a start guid? If so, how do they specify where to "end", e.g. when things started working again. 

The alternative would be for the script to actually die when there's an error by re-raising the exception and providing output so the user knows what batch failed and what the last successful GUID was